Framework/Core/include/Framework/AnalysisHelpers.h

```diff
@@ -483,14 +540,16 @@ struct Service {
   }
 };
 
-template <typename T>
+auto getTableFromFilter(soa::is_filtered_table auto const& table, soa::SelectionVector&& selection)
+{
+  return std::make_unique<o2::soa::Filtered<std::decay_t<decltype(table)>>>(std::vector{table}, std::forward<soa::SelectionVector>(selection));
+}
+
+template <soa::is_table T>
+  requires(!soa::is_filtered_table<T>)
```

Comment:
Do you really need the requires? It's simply the negation of the above, no?

Reply:
`is_table` includes a `is_filtered_table` and compiler considers the function call ambiguous without this. I have  defined `is_not_filtered_table` in #13679